app/components/workflow-grants.js

```diff
@@ -201,7 +204,7 @@ export default Component.extend({
         this.get('store').findRecord('grant', event.target.value).then((g) => {
           g.get('primaryFunder.policy'); // Make sure policy is loaded in memory
           this.addGrant(g);
-          Ember.$('select')[0].selectedIndex = 0;
+          $('select')[0].selectedIndex = 0;
```

Comment:
Does not need to be addressed now, but we should note this to be changed to not use jQuery eventually